scripts/build.py

```diff
@@ -285,10 +361,19 @@ def get_font_filepaths(dependency_bundle, dependency_dir):
         list(str). List of paths to font files that need to be copied.
     """
     if 'fontsPath' not in dependency_bundle:
+        # Skipping dependency bundle in manifest.json that does not have
```

Comment:
The different tense in the various comments is a bit confusing. E.g. "Skipping" vs "Obtain / Walk". 

Let's try to make it consistent throughout -- probably the latter is best. (So, "Skipping" --> "Skip".) Also, "bundle" --> "bundles" and "does --> "do".

Reply:
Fixed grammar inconsistency. Thanks!